docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Routing Access Requests to Access Plugins
+description: How to set up Teleport's Access Monitoring Rules
+---
+
+With Teleport's Access Monitoring Rules Access Request notifications
+can be routed to plugins based on based on several conditions.
+For example, if you wish to notify different recipients e.g Slack channels,
+based on what roles or resources are being accessed, or based on what user
+is making the request.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A managed Teleport Enterprise account.
+
+- The Enterprise `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
+
+  You can verify the tools you have installed by running the following commands:
+
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  You can download these tools by following the appropriate [Installation 
+  instructions](../../installation.mdx#linux) for your environment and Teleport edition.
+
+- At least one of the Teleport Access Plugins that support using being set up.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Create an Access Monitoring Rule
```

Comment:
Can you split this section into 2: creating via UI and creating via a YAML resource (UI or CLI). I would have a structure something like this:

```
## Step 1/2. Create an Access Monitoring Rule

You can define an access request notification rule in two ways: using a web UI dialog or by creating a YAML resource file containing the definition of a rule.

### Creating in web UI

// Here explain where to go in the UI, show screenshots.

### Creating using YAML resource

For more advanced scenarios, you can define a notification rule in raw YAML resource format, which can then be created by both web UI, or the CLI.

// Rest of your explanation below follows.
```

Later we can add Terraform and Kube Operator here as well.